README.md

```diff
@@ -1,2 +1,108 @@
-# doppler-html-editor-api
-API Rest from Doppler HTML Editor
+# Doppler HTML Editor API
+
+## Campaigns
+`GET /accounts/{accountName}/campaigns/{campaignId}/content/design`
+
+Response:
+```
```

Comment:
You can take advantage of syntax highlighting
 
```suggestion
\```json
```

Reply:
Fixed!